lib/server-helpers.js

```diff
@@ -119,10 +119,18 @@ var addRoutes = function(app, routes, data) {
   return _routes;
 };
 
+var httpAuth = require('http-auth');
+var auth = function(creds) {
+  return httpAuth.connect(httpAuth.basic({}, function(user, pass, cb) {
+    return cb(user === creds.username && pass === creds.password);
```

Comment:
For this particular line, I think using extra parentheses and being explicit about order of operations would be for the best.